eng/pipelines/libraries/stress/http.yml

```diff
@@ -40,6 +40,7 @@ jobs:
     displayName: Build CLR and Libraries
   
   - bash: |
+      exit 1
```

Comment:
???

Reply:
This should be WIP right now, I'm testing something Santi suggested.